### ISSUE TITLE
Does the addin analyze the entire email thread?

### ISSUE THREAD

**Issue by shaggy2626** [qmatteoq/outlook-businessmails-openai#2]
Will the add work for replies to email? For instance if there is an entire email thread will my input be based of the event thats took place on the email thread?

**Comment by qmatteoq** [comment]
No, this first version only takes the text you type into the panel and turn into a business mails. I haven't added yet the capability to reply based on the context of the mail.

**Comment by shaggy2626** [comment]
I installed the raw manifest file from the repo through the webapp MSFT page for outlook add ons. When I open the add on in outlook it says connection issue and not able to resolve a host. Is there something I need to change?

**Comment by shaggy2626** [comment]
> No, this first version only takes the text you type into the panel and turn into a business mails. I haven't added yet the capability to reply based on the context of the mail.

I installed the raw manifest file from the repo through the webapp MSFT page for outlook add ons. When I open the add on in outlook it says connection issue and not able to resolve a host. Is there something I need to change?

**Comment by shaggy2626** [comment]
![image](https://user-images.githubusercontent.com/120447538/225178509-8098c854-c5b2-4e03-9bfa-1da9fb489331.png)
